erpnext/crm/utils.py

```
@@ -261,3 +267,31 @@ def delete_note(self, row_id):
+	# Convert date to moment.js format
+	moment_format = date_format.replace("yyyy", "YYYY").replace("mm", "MM").replace("dd", "DD")
+	
+	# Create moment object and format according to user's locale
+	try:
+		if isinstance(date, str):
+			# Parse the date string
+			parsed_date = frappe.utils.getdate(date)
+			formatted_date = frappe.utils.momentjs.moment(parsed_date).locale(user_language).format(moment_format)
+		else:
+			formatted_date = frappe.utils.momentjs.moment(date).locale(user_language).format(moment_format)
+		return formatted_date
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)